### PR TITLE
Add file commands

### PR DESCRIPTION
**Pull request by rodrigotuna**
This PR adds commands for interacting with files while the simulation is running.

Commands:
- `inductiva tasks ls <task_id>`
image

- `inductiva tasks tail <task_id> <filename>`
image

### CODE REVIEW

**Review by Pbarbosa92 — CHANGES_REQUESTED** [review]
After testing I've changed my mind.

We need to change some names. We already have `inductiva tasks list`it would make sense to me that `inductiva tasks ls` would do the same as the list.

Maybe this could be `inductiva tasks track list <id>`  and `inductiva tasks track tail <id> <filename>`. Not sure about this names but having list/ls in two places is confusing.

**Comment by rodrigotuna** [comment]
> After testing I've changed my mind.
> 
> We need to change some names. We already have `inductiva tasks list`it would make sense to me that `inductiva tasks ls` would do the same as the list.
> 
> Maybe this could be `inductiva tasks track list <id>` and `inductiva tasks track tail <id> <filename>`. Not sure about this names but having list/ls in two places is confusing.

Would you agree with something like `inductiva tasks list_dir <id>`? Otherwise it would be too lengthy.

**Review by Pbarbosa92 — APPROVED** [review]

File: inductiva/_cli/cmd_tasks/tail.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import argparse
+import sys
+import asyncio
+import logging
+
+from inductiva import _cli, tasks
+
+
+def tail(args: argparse.Namespace, fout: TextIO = sys.stdout):
+    logging.getLogger().setLevel(level=logging.WARNING)
```
[the remark this answers is not in the document]
Comment (rodrigotuna) [reply inductiva/_cli/cmd_tasks/tail.py:12]: Aiortc is too verbose in level=INFO

File: inductiva/tasks/file_tracker.py [hunk 2]
```diff
@@ -3,12 +3,12 @@
...
 import uuid
 import enum
-from aiortc import RTCPeerConnection, RTCSessionDescription, RTCConfiguration, RTCIceServer
+import aiortc
 
 # STUN/TURN server configuration
 ICE_SERVERS = [
-    RTCIceServer("stun:34.79.246.4:3478"),
-    RTCIceServer("turn:34.79.246.4:3478")
+    aiortc.RTCIceServer("stun:34.79.246.4:3478"),
```
Comment (hpenedones) [inline inductiva/tasks/file_tracker.py:10]: I will now create a DNS entry for webtrc.inductiva.ai for this IP address...

**Pull request merged**